pkg/exchange/okex/exchange.go

```diff
@@ -339,3 +339,30 @@ func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval type
 	return klines, nil
 
 }
+
+func (e *Exchange) QueryOrder(ctx context.Context, q types.OrderQuery) (*types.Order, error) {
+	if len(q.Symbol) == 0 {
+		return nil, errors.New("okex.QueryOrder: InstrumentID is required parameter")
```

Comment:
fixed error object could be pulled out to:

```
var ErrXXX = errors.New("...")
```

Reply:
Updated, I declare a error type variable in exchange.go